lib/tests/assertions.rb

```diff
@@ -15,6 +15,75 @@ def assert_equal(expected, actual, message="", data="")
         end
       end
 
+      def assert_operator(operator, expected, actual, message="", data="")
+        case operator
+        when :equals
+          unless assertion_negated( expected == actual )
+            message += " Expected: #{expected}, but found: #{actual}."
+            raise AssertionException.new message, data
+          end
+        when :notEquals
+          unless assertion_negated( expected != actual )
+            message += " Expected not: #{expected}, but found: #{actual}."
+            raise AssertionException.new message, data
+          end
+        when :in
+          unless assertion_negated(expected.split(",").include?(actual))
+            message += " Expected #{actual} to be in #{expected}."
+            raise AssertionException.new message, data
+          end
+        when :notIn
+          unless assertion_negated(!expected.split(",").include?(actual))
+            message += " Expected #{actual} to not be in: #{expected}."
+            raise AssertionException.new message, data
+          end
+        when :greaterThan
+          unless assertion_negated(actual > expected)
+            message += " Expected #{actual} to be greater than #{expected}."
+            raise AssertionException.new message, data
+          end
+        when :lessThan
+          unless assertion_negated(actual < expected)
+            message += " Expected #{actual} to be greater than #{expected}."
+            raise AssertionException.new message, data
+          end
+        when :empty
+          unless assertion_negated(actual.nil? || actual.length == 0)
+            message += " Expected #{actual} to be empty."
+            raise AssertionException.new message, data
+          end
+        when :notEmpty
+          unless assertion_negated(!actual.nil? && actual.length > 0)
+            message += " Expected #{actual} to not be empty."
+            raise AssertionException.new message, data
+          end
+        when :contains
+          unless assertion_negated(actual.include?(expected))
+            message += " Expected #{actual} to contain #{actual}."
+            raise AssertionException.new message, data
+          end
+        when :notContains
+          unless assertion_negated(!actual.include?(expected))
+            message += " Expected #{actual} to not contain #{actual}."
+            raise AssertionException.new message, data
+          end
+        else
+          message += " Invalid test; unknown operator: #{operator}."
+          raise AssertionExection.new message, data
+        end
+
+      end
+
+      def assert_valid_profile(response, klass)
+        unless assertion_negated( response[:code].to_s == "200")
+          
+          raise AssertionException.new "Server created a #{klass.name.demodulize} with the ID `_validate` rather than validate the resource." if response[:code].to_s == "201"
+
+          raise AssertionException.new "Response code #{response[:code]} with no OperationOutcome provided"
```

Comment:
Yeah this code was just taken from what is in the other tests.  would you like me to fix?  I probably should fix the other tests too then...